sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceActionCancelEventData.java

```diff
@@ -243,18 +276,44 @@ public ResourceActionCancelEventData setAuthorization(String authorization) {
      * Get the claims property: The properties of the claims.
      *
      * @return the claims value.
+     * @deprecated This method is no longer supported since v4.9.0.
+     * <p> Use {@link ResourceActionCancelEventData#getResourceClaims()} instead.
```

Comment:
If this was manually added, can you please file a GitHub issue to update customization code to do this automatically when we regenerate code next time?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/26882